### PR TITLE
Updated Bar examples to be carbon-centric

### LINKED ISSUE #165: Update documentation for Bar component to be carbon-centric
# Story

## Description

As a current consumer of carbon, I'd like for the public facing documentation for the Bar component to utilize carbon until terra-graphs is released

## Motivation

So that I will have the information I need to use the Bar component from carbon as things currently are released

## Definition of Done

We'll know we're done when the documentation in terra-graphs-docs is utilizing carbon instead of terra-graphs.

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

Closes #165 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/TimeseriesWithEventline.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```
Comment (AshishMotanamGurunadham) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/TimeseriesWithEventline.jsx:1]: Looks like eventline example is not working fine. Can you please take a look at it?

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/Reflow/SimplePanning.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        break;
+      case 'panRight':
+        hour = panState.initial + panState.factor;
+        break;
+      default:
+        return panState;
+    }
+
+    graph.config.axis.x.lowerLimit = new Date(2016, 0, 1, hour).toISOString();
+    graph.config.axis.x.upperLimit = new Date(2016, 0, 2, hour).toISOString();
```
Comment (tbiethman) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/Reflow/SimplePanning.jsx:46]: See my related comment regarding side-effects inside reducer functions: https://github.com/cerner/terra-graphs/pull/193#discussion_r679991504

https://codeartistry.io/what-is-a-reducer-in-javascript-f4fd40f98324

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/TimeseriesWithEventline.jsx (hunk 1 above)
Comment (sdadn) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/TimeseriesWithEventline.jsx:1]: fixed here: 54816259a1f0c2401d421315c02ef4dfae784e0f

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/Reflow/SimplePanning.jsx (hunk 2 above)
Comment (sdadn) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/Reflow/SimplePanning.jsx]: Fixed here: 8b9a77697c7fcea954b92104e778a7f742398301
(the no newline error was fixed in a subsequent commit)

**Review by AshishMotanamGurunadham — APPROVED** [review]

**Review by tbiethman — APPROVED** [review]

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/AxisInfoTextLabels.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import '@cerner/terra-graphs/lib/components/Graph.module.scss';
+import '@cerner/terra-graphs/lib/components/Bar/BarGraph.module.scss';
```
Comment (mjhenkes) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/AxisInfoTextLabels.jsx:6]: The terra-graphs package won't be release when we pull terra-grachs-docs into terra-ui, so any references to terra-graphs will fail.

Comment (sdadn) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/AxisInfoTextLabels.jsx:6]: I created a new issue to address this:  #202

**Pull request merged**